.github/workflows/pipeline-deploy-app-test.yaml

```diff
@@ -107,6 +108,7 @@ jobs:
       - name: Build and deploy application code
         run: |
           export SERVICE_NAME=${{ matrix.service-name }}
+          export DEPLOYMENT_WORKSPACE=${{ needs.derive-deployment-workspace.outputs.deployment_workspace }}
```

Comment:
I think that this also needs adding to pipeline-deploy-app-test.yaml (see error when deploying when merging to main `An error occurred (ResourceNotFoundException) when calling the UpdateFunctionCode operation: Function not found: arn:aws:lambda:eu-west-2:***:function:organisations-data-manager-default) `